src/edfi-paging-test/.gitignore

```diff
@@ -0,0 +1,2 @@
+.coverage
+*.xml
```

Comment:
Should the detail and summary reports be ignored as well? If full path is not given for output directory, reports get created at the repo root.